Lib/multiprocessing/context.py

```diff
@@ -199,15 +202,28 @@ def get_start_method(self, allow_none=False):
     def set_start_method(self, method, force=False):
         raise ValueError('cannot set start method of concrete context')
 
-    @property
-    def reducer(self):
+    def get_reducer(self):
         '''Controls how objects will be reduced to a form that can be
         shared with other processes.'''
-        return globals().get('reduction')
+        reducer = getattr(self, "_reducer", None)
```

Comment:
You can add `_reducer = None` in the class definition to avoid the `getattr` call (and make bugs less likely).